recipes/xo/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.11" %}
+
+package:
+  name: xo
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/scopatz/xo.git
+  git_tag: 8f486a2709d376d00f98c09a86179bce2a99099a
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  run:
+    - python >=3.4
```

Comment:
We need to be super careful here - conda does not do what you (or I) would expect. Sure, conda will decide to build with a ```python >=3.4```, but the Python that is used to ```build``` the package is *not* necessarily the python that is used to ```run``` the package.
Adding another line which states ```python x.x``` is necessary (I think both in the run and in build sections, but you can test that by looking at the build string and observing a ```py34```/```py35``` suffix)